map.css

```diff
@@ -31,13 +31,19 @@ body {
     font-size: normal;
 }
 
-h1, h2 {
-    margin: 10px;
+h1, h2, h3 {
+    margin: 10px;    
+}
+
+h1 {
+    font-size: 21px;
```

Comment:
Absolute font sizes are not a good idea here. For portability and consistency I suggest to go for `x-large`.